### ISSUE TITLE
Console IO

### ISSUE THREAD

**Issue by DCNick3** [DCNick3/uwin#21]
For now, only basic support without precise semantics:
- [ ] `WriteConsole` (stdout and stdin)
- [ ] `ReadConsole` (might block the thread, so depends on #3 & #20)

[Low-level functions](https://docs.microsoft.com/en-us/windows/console/low-level-console-i-o) will require to implement an actual screen buffer, which is not needed yet

**Comment by DCNick3** [comment]
Looking at [wine's design](https://wiki.winehq.org/Wine_User%27s_Guide#Text_mode_programs_.28CUI:_Console_User_Interface.29) might be a good idea.

Currently planned features are like what wine's `bare streams` do.
